solutions/cses-1196.mdx

```diff
@@ -1,3 +1,77 @@
+Solution 1:
```

Comment:
Move ```Solution 1:``` to after the section with the problem information (id, source, title, author)